website/docs/r/codebuild_project.html.markdown

```diff
@@ -150,6 +150,10 @@ resource "aws_codebuild_project" "example" {
     type            = "GITHUB"
     location        = "https://github.com/mitchellh/packer.git"
     git_clone_depth = 1
+
+    git_submodules_config {
+        fetch_submodules = "true"
```

Comment:
Nit: Since this argument is a boolean, we should prefer the unquoted identifier here in the example 👍 

```suggestion
        fetch_submodules = true
```

Reply:
Definitely!